src/ls_instructions/ls_instructions_from_gates.cpp

```diff
@@ -12,12 +12,7 @@ std::queue<LSInstruction> LSIinstructionFromGatesGenerator::make_t_gate_instruct
     // Currently we ignore the is_dagger argument, since the procedure in this case is the same
     std::queue<LSInstruction> next_instructions;
     PatchId new_magic_state_id = id_generator_.new_id();
-<<<<<<< HEAD
-    next_instructions.push({.operation={MagicStateRequest{new_magic_state_id, target_id}}, .wait_at_most_for=MagicStateRequest::DEFAULT_WAIT});
-=======
-    
-    next_instructions.push({.operation={MagicStateRequest{new_magic_state_id}}, .wait_at_most_for=MagicStateRequest::DEFAULT_WAIT, .clients={target_id}});
->>>>>>> chris/wave
+    next_instructions.push({.operation={MagicStateRequest{new_magic_state_id, target_id}}, .wait_at_most_for=MagicStateRequest::DEFAULT_WAIT, .clients={target_id}});
```

Comment:
I apologize for this. I must have not staged right. This is the challenge of pushing without CI